SimPlatform/dashboard.cpp

```diff
@@ -0,0 +1,20 @@
+#include "dashboard.h"
+#include "ui_dashboard.h"
+#include <QMessageBox>
+
+DashBoard::DashBoard(QWidget *parent) :
```

Comment:
Could you rename Dashboard in GpaView? This is not a dashboard per se. It can be used to create a dashboard.